### PR TITLE
rename item classification

### LINKED ISSUE #1034: Rename itemClassificationScheme.csv to classificationScheme.csv
Follow-up to #982

### PR DESCRIPTION
**Pull request by yolile**
closes #1034

### CODE REVIEW

File: docs/schema/codelists.md [hunk 1]
```diff
@@ -40,16 +40,14 @@ The organizations, economic operators or other participants in a contracting pro
 :file: ../../build/current_lang/codelists/partyRole.csv
 ```
 
-### Item Classification Scheme
+### Classification Scheme
 
-Items should be classified using existing item classification schemes, such as the [EC Common Procurement Vocabulary (CPV)](https://simap.ted.europa.eu/web/simap/cpv).
```
Comment (yolile) [inline docs/schema/codelists.md:45]: @jpmckinney I also moved this sentence from here to the item.classification definition in the release-schema.

**Comment by yolile** [comment]
Not sure if I should commit the locales too

File: schema/release-schema.json [hunk 2]
```diff
@@ -1405,7 +1405,7 @@
         },
         "classification": {
           "title": "Classification",
-          "description": "The primary classification for the item.",
+          "description": "The primary classification for the item. Items should be classified using existing item classification schemes, such as the [EC Common Procurement Vocabulary (CPV)](https://simap.ted.europa.eu/web/simap/cpv).",
```
Comment (jpmckinney) [inline schema/release-schema.json]: I think this sentence was unnecessary in the original documentation, and we can just remove it. Any scheme used by publishers exists (it's not like they invented one just for OCDS).

**Review by jpmckinney — COMMENTED** [review]
No need to update the PO files. We're going to have a lot to translate in 1.2, so it's not really worth the effort to edit the source and translated texts.

**Pull request merged**